source/ShipInfoDisplay.cpp

```diff
@@ -192,8 +192,8 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 		attributeValues.push_back(Format::Number(attributes.Get("hull")));
 	}
 	attributesHeight += 20;
-	double emptyMass = attributes.Mass();
-	double currentMass = ship.Mass();
+	double emptyMass = attributes.Mass() / (1. + attributes.Get("inertia reduction"));
+	double currentMass = ship.InertialMass();
```

Comment:
I dont know, but maybe just display inertial mass and normal mass?
I personally would like to know both

Reply:
oc only if inertia reduction is present

Reply:
to see how much effect the inertia has

Reply:
I'm fine with it like this, dont want to give too much information

Reply:
The effects of your inertial mass will be felt in your acceleration and turn rate, but your normal mass still has an impact on other things. As such, I think that we should still display your normal mass here, and only the acceleration and turning values printed farther down from here should take your inertial mass into effect.

Reply:
The effects of your inertial mass will be felt in your acceleration and turn rate, but your normal mass still has an impact on other things. As such, I think that we should still display your normal mass here, and only the acceleration and turning values printed farther down from here should take your inertial mass into effect.